src/service-override/scm.ts

```diff
@@ -0,0 +1,16 @@
+import { IEditorOverrideServices } from 'vs/editor/standalone/browser/standaloneServices'
+import { ISCMService, ISCMViewService } from 'vs/workbench/contrib/scm/common/scm'
+import { SCMService } from 'vs/workbench/contrib/scm/common/scmService'
+import { SCMViewService } from 'vs/workbench/contrib/scm/browser/scmViewService'
+import { IQuickDiffService } from 'vs/workbench/contrib/scm/common/quickDiff'
+import { QuickDiffService } from 'vs/workbench/contrib/scm/common/quickDiffService'
+import 'vs/workbench/contrib/scm/browser/scm.contribution'
+import { SyncDescriptor } from '../services'
```

Comment:
Can you please import it directly from the right vscode file?

Reply:
Oh right! I keep forgetting that!